config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
```

Comment:
If this file is required, we need to add one in `openshift/system/config`

But I'd say we don't need to enable it (i.e. don't use redis)

Reply:
Maybe rails automatic update added it. We better get rid of it.

Reply:
As far as I know, that file is for configuring `ActionCable`, which we removed because it's not being used. See: https://github.com/3scale/porta/pull/3339/files#diff-c1fd91cb1911a0512578b99f657554526f3e1421decdb9e908712beab57e10f9R12-R29

I'm removing it.

Reply:
Oh even better!
I removed it previously from the config examples https://github.com/3scale/porta/pull/3225/commits/a4947ffeabfe3befa8fbc06c07442108a23433bc#diff-0c400262ffebf26a5f82d65ac15697731ec1917292851cda09a9687ed8bd1559

But I thought that maybe it's required now in Rails 6. Great that we just disable action cable :+1: